lib/pbench/test/functional/agent/cli/commands/tools/test_clear_tools.py

```diff
@@ -0,0 +1,185 @@
+import pytest
+
+
+def test_clear_tools_help():
+    command = ["pbench-clear-tools", "--help"]
+    out, err, exitcode = pytest.helpers.capture(command)
+    assert b"Usage: pbench-clear-tools [OPTIONS]" in out
+    assert exitcode == 0
+
+
+def test_clear_tools_test12(monkeypatch, agent_config, pbench_run, pbench_cfg):
+    monkeypatch.setenv("_PBENCH_AGENT_CONFIG", str(pbench_cfg))
+    default_group = pbench_run / "tools-v1-default" / "testhost.example.com"
+    default_group.mkdir(parents=True)
+    foo_group = pbench_run / "tools-v1-default" / "fubar"
+    foo_group.mkdir(parents=True)
+    mpstat = default_group / "mpstat"
+    mpstat.touch()
+
+    command = ["pbench-clear-tools"]
+    err, out, exitcode = pytest.helpers.capture(command)
+    assert b'All tools removed from host, "testhost.example.com"' in out
+    assert exitcode == 0
+    assert mpstat.exists() is False
+    assert default_group.exists() is False
+    assert foo_group.exists() is False
+
+
+def test_clear_tools_test13(monkeypatch, agent_config, pbench_run, pbench_cfg):
+    monkeypatch.setenv("_PBENCH_AGENT_CONFIG", str(pbench_cfg))
+    default_group = pbench_run / "tools-v1-default" / "testhost.example.com"
+    default_group.mkdir(parents=True)
+    default_mpstat = default_group / "mpstat"
+    default_mpstat.touch()
+    foo_group = pbench_run / "tools-v1-default" / "fubar2"
+    foo_group.mkdir(parents=True)
+    foo_mpstat = foo_group / "mpstat"
+
+    command = ["pbench-clear-tools", "--remote=fubat2"]
```

Comment:
Looks like the output from this test does not match the original.  Could that be because we have `fubat2` here instead of `fubar2`?